blockchain/state/state_object.go

```diff
@@ -257,15 +257,26 @@ func (self *stateObject) updateStorageTrie(db Database) Trie {
 	return tr
 }
 
-// UpdateRoot calls updateStorageTrie to get the latest root hash of the object's storage trie
-// and sets the storage trie root to the newly updated one.
+// updateStorageRoot sets the storage trie root to the newly updated one.
```

Comment:
Is this function used elsewhere? If not, what about removing this function?

Reply:
It is used at line 850 of `statedb.go`